### DISCUSSION TITLE
Reserved environment variables

### DISCUSSION THREAD

**Discussion by meel-hd** [vercel/vercel#9136] [Ideas]
Deploying to production using the command `vercel --prod` which picks my current local .env file and it doesn't warn or prompt that some of the name of environment variables are  [reserved](https://vercel.com/docs/concepts/limits/overview?query=reserve#reserved-variables). which I didn't know anything about this reserved variable names if I didn't past them in the **project settings  Environment variables.**

**Comment by amyegan** [comment]
Hi @meel-hd! Sorry for the delay. Are you asking for a workaround, or suggesting a CLI change to include a warning about reserved env variables? Or maybe both? 😄  I want to be sure I get you what you need for this issue

**Reply by meel-hd** [reply]
Yes I think that would be perfect,  warning that an environment variable is not going to be picked up to the Vercel deployment is better than nothing, that results in a broken app without knowing that is related to Vercel's reserved environment variables.

**Reply by amyegan** [reply]
Awesome! I'll mark this as a feature suggestion and share it with the team
